### ISSUE TITLE
Add documentation and code examples for multi key vault setups

### ISSUE THREAD

**Issue by spowser** [Azure/AppConfiguration#666]
I could not find any documentation on the proper way to register multiple key vaults that support a given app configuration setup.  This seemed to be the solution:

```
configBuilder.AddAzureAppConfiguration(options =>
{
    var azureCredentials = new DefaultAzureCredential();
    options.Connect(new Uri("https://someurltoappconfig"), azureCredentials).Select("WhateverPrefix:*", "WhateverLabel")
           .ConfigureKeyVault(keyVaultOptions => keyVaultOptions
                   .Register(new SecretClient(new Uri(azureVaultUrl), azureCredentials))
                   .Register(new SecretClient(new Uri(anotherAzureVaultUrl), azureCredentials)))
}
```

Can we add something around this topic to the documentation so others don't have to trial and error?

**Comment by zhenlan** [comment]
Hi @spowser if all your Key Vaults have been granted permission to be accessed by the same credential, you can call the API below once and you don't have to call it for every Key Vault.

```cs
keyVaultOptions.SetCredential(azureCredentials)
```

More details of the API reference can be found at
https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.azureappconfiguration.azureappconfigurationkeyvaultoptions.setcredential?view=azure-dotnet

**Comment by spowser** [comment]
@zhenlan That works great.  I just checked the documentation again and [this suggests](https://docs.microsoft.com/en-us/azure/azure-app-configuration/use-key-vault-references-dotnet-core?tabs=core5x#update-your-code-to-use-a-key-vault-reference) your solution.  Tried to find where the .Register() documentation came from so I think we are ok to close this.  My only suggestion is that step 4 of the noted document spell out that the credentials will work for any key vaults being accessed.  It currently talks about Key Vault in a singular sense so maybe some clarification is warranted.
